deployment_scripts/v2.4/upgrade.sh

```diff
@@ -107,14 +104,8 @@ spec:
     command: ["sh", "-c", "wget https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64", "-o", "/usr/bin/yq", "&&", "chmod", "+x", "/usr/bin/yq"]
```

Comment:
I forgot this line yesterday. This is not used anymore since we removed the InitContainer. You can delete it too